index.xml

```diff
@@ -734,6 +734,21 @@ NOTE: The most up to date place for developers to find the table above is
         <t>Status: standard</t>
 
     </section>
+
+    <section anchor="mh-ni-hash-algorithm" title="The 'mh' Named Information Hash Algorithm">
+      <t>
+        This memo registers the "mh" hash algorithm in the
+        <eref target="https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg">Named Information Hash Algorithm</eref>
+        registry with the following values:
+      </t>
+
+        <t>ID: 49</t>
```

Comment:
Any particular reason the number 49 was chosen?

Reply:
Mostly just symbolic – the number 49 is the identification of [multihash](https://github.com/multiformats/multicodec/blob/master/table.csv) (0x31) so it's a sort of a nod to that. Perhaps it could also allow for some compatibility between the RFC 6920 binary format and a potential `<multiformat id = multihash (0x31)><value>` identifier format.

Reply:
I love it! Thanks, merging.